vendor/github.com/docker/libnetwork/drivers/windows/windows.go

```diff
@@ -440,6 +448,31 @@ func parseEndpointOptions(epOptions map[string]interface{}) (*endpointConfigurat
 	return ec, nil
 }
 
+func parseEndpointConnectivity(epOptions map[string]interface{}) (*endpointConnectivity, error) {
```

Comment:
This pattern of using `map[string]interface{}` as a struct in multiple places is worrying.